src/main/java/dev/streamx/cli/util/ExceptionUtils.java

```diff
@@ -0,0 +1,15 @@
+package dev.streamx.cli.util;
+
+public final class ExceptionUtils {
+  public static RuntimeException sneakyThrow(Throwable t) {
+    if (t == null) {
+      throw new NullPointerException("t");
```

Comment:
should we pass a string in a constructor?

Reply:
It's copy-pasted from [Lombok](https://github.com/projectlombok/lombok/blob/master/src/core/lombok/Lombok.java#L28)

As we do not use Lombok in our project (maybe we should?) I 'imported' it's utils mechanism allowing to throw checked exceptions as uncheck (as Sneaky throws it's well known mechanism and widely used)